db/migrate/20170427011934_re_add_owner_to_application.rb

```diff
@@ -0,0 +1,7 @@
+class ReAddOwnerToApplication < ActiveRecord::Migration[5.0]
+  def change
+    add_column :oauth_applications, :owner_id, :integer, null: true
+    add_column :oauth_applications, :owner_type, :string, null: true
```

Comment:
Wait, does this have to be polymorphic? We're never gonna attach the application to anything other than `User`. If this could be only owner_id with a foreign key ensuring integrity with the users table, that'd be far better.